### DISCUSSION TITLE
Where to get latest patches?

### DISCUSSION THREAD

**Discussion by TumbleFur** [Mr-MIBonk/M.I.B._More-Incredible-Bash#609] [Q&A] [answered]
Hi guys

Sorry if this has been asked before, but I haven't been able to find anything:
Where do I find the latest patches? The two links in the Readme seem to be broken.

Thanks

**Comment by DanielForsell** [comment, marked as answer]
Hi,

I found it under the first tab in mibsolutions (mib -> M.I.B -> Patches)
Here also direct link https://mibsolution.one/#/1/18

**Comment by TumbleFur** [comment]
D'oh... Thanks, looks like site was just unavailable for a moment when I tried, so I thought it was offline for good.
Sorry for bothering everyone.
Cheers
